data/hai outfits.txt

```diff
@@ -114,7 +114,7 @@ outfit "Railgun"
 		"hit force" 44
 		"stream"
 		"submunition" "rslug" 1
-	description "The Hai created this weapon for their Flea drones, providing mobile long range support for their close quarters Pond Strider carrier. This particular incarnation of a rail weapon imparts a massive static electric charge to the projectile's flux capacitor, and motivates the slug with electric repulsion. Once in flight, the projectile's charge will bend trajectory very slightly, and allows it to damage the hull a little bit through shields."
+	description "The Hai created this miniaturized incarnation of the classic railgun for their Flea drones, providing mobile long range support for the close quarters Pond Strider carrier."
```

Comment:
I wouldn't consider subatomic particles as slugs or even projectiles at that. In essence, the Particle Cannon seems like a weaponized particle accelerator. The mechanics of projectile acceleration are the same, but I wouldn't call it a railgun.

Okay, let's assume that the Particle Cannon and Proton Gun are "railguns". Then the new railgun shouldn't even be called a railgun, because there are significant differences between them.